src/ibex_bluesky_core/utils.py

```diff
@@ -104,3 +109,63 @@ def calculate_polarisation(
     polarisation_value.variances = variance_return
 
     return polarisation_value
+
+
+def calculate_erf_stretch(
```

Comment:
Do we need to expose this publically, or should it just be a private helper method within `_fitting.py`?